src/ComposerRequireChecker/GeneratorUtil/ComposeGenerators.php

```diff
@@ -2,11 +2,11 @@
 
 namespace ComposerRequireChecker\GeneratorUtil;
 
-use Generator;
+use Traversable;
 
 final class ComposeGenerators
 {
-    public function __invoke(Generator ...$generators): Generator
+    public function __invoke(Traversable ...$generators): Traversable
```

Comment:
This could be `Traversable<TKey, TValue>` as input, and `Traversable<int, TValue>` as output